core/identity/src/autoconf.rs

```diff
@@ -0,0 +1,37 @@
+use std::env;
+
+use ethsign::*;
+use rustc_hex::FromHex;
+
+use ya_core_model::NodeId;
+
+use crate::id_key::IdentityKey;
+use anyhow::Context;
+
+// autoconfiguration
+const ENV_AUTOCONF_PK: &str = "YAGNA_AC_IDENTITY_PK";
```

Comment:
Naming consistency would be better if you use eg.
```suggestion
const ENV_AUTOCONF_PK: &str = "YAGNA_AUTOCONF_ID_PRIV_KEY";
```